src/main/java/edu/iis/powp/command/RectangleComplexCommandFactory.java

```diff
@@ -0,0 +1,26 @@
+package edu.iis.powp.command;
+
+public class RectangleComplexCommandFactory {
```

Comment:
PLEASE FIX: Factory is not tested. You can add a test to menu in TestPlotterApp.